### PR TITLE
Updates to HTAN Notebooks

### PR DESCRIPTION
**Pull request by clarisse-lau**
- Replace BQ table queried in Building_AnnData_with_Subset_of_Cells_from_BQ.ipynb and Investigating_Single_Cell_HTAN_Data.ipynb notebooks after original table was removed from ISB-CGC due to presence of PHI. 
- Add details of nearest neighbors example to Explore_HTAN_Spatial_Cellular_Relationships.Rmd

### CODE REVIEW

**Comment by clarisse-lau** [comment]
(pinging @lhagen-isb as I'm unable to request reviewers :) )

**Comment by clarisse-lau** [comment]
Hi @lhagen-isb @boaguilar ! 
Just wanted to check in here- are there any modifications you'd like us to make in order to get this merged?

File: HTAN/Python Notebooks/Investigating_Single_Cell_HTAN_Data.ipynb [hunk 1]
```diff
@@ -49,7 +49,7 @@
     {
       "cell_type": "markdown",
       "source": [
-        "[![Open in Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/isb-cgc/Community-Notebooks/blob/master/HTAN/Python%20Notebooks/Investigating_Single_Cell_HTAN_Data.ipynb)"
+        "[![Open in Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/ncihtan/Community-Notebooks/blob/update-single-cell-python-notebook/HTAN/Python%20Notebooks/Investigating_Single_Cell_HTAN_Data.ipynb)"
```
Comment (boaguilar) [inline HTAN/Python Notebooks/Investigating_Single_Cell_HTAN_Data.ipynb]: Just checking here, should this be isb-cgc instead of ncihtan?

Comment (clarisse-lau) [reply HTAN/Python Notebooks/Investigating_Single_Cell_HTAN_Data.ipynb]: Good catch! changed to master branch of isb-cgc
(cc @PozhidayevaDarya )

Comment (PozhidayevaDarya) [reply HTAN/Python Notebooks/Investigating_Single_Cell_HTAN_Data.ipynb]: Thank you!!

**Pull request merged**